crispy_forms/layout.py

```diff
@@ -290,7 +290,7 @@ def render(self, form, form_style, context, template_pack=TEMPLATE_PACK, **kwarg
 
         legend = ""
         if self.legend:
-            legend = "%s" % Template(str(self.legend)).render(context)
+            legend = f"{Template(str(self.legend)).render(context)}"
```

Comment:
I'm not at all sure that this, as an example, is actually easier to read.

Reply:
Agree. Some are far easier, but this one is not great - it's hard before, its even worse now.